src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1957,6 +1989,41 @@ public partial struct Single : System.IComparable, System.IComparable<float>, Sy
         public static bool TryParse(string s, System.Globalization.NumberStyles style, System.IFormatProvider provider, out float result) { throw null; }
         public static bool TryParse(string s, out float result) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct Span<T>
+    {
+        public static Span<T> Empty { get { throw null; } }
+        public Span(T[] array) { throw null; }
+        public Span(T[] array, int start) { throw null; }
+        public Span(T[] array, int start, int length) { throw null; }
+        [CLSCompliant(false)]
```

Comment:
The resulting IL is identical: the C# language allows the "Attribute" suffix to be omitted on attributes, and we're in the System namespace here so it's also superfluous.  I didn't see that the style throughout the rest of the file was to fully spell everything out, but to match that I probably should have done so.  I'll merge this and then fix it in the other System.Runtime changes I have pending, so as not to need to restart CI for this.